packages/lambda-xyz/src/__test__/xyz.test.ts

```diff
@@ -175,8 +171,8 @@ o.spec('LambdaXyz', () => {
             );
 
             const vdom = await VNodeParser.parse(body);
-            const url = vdom.tags('ResourceURL').next().value!;
-            o(url.toString()).equals(
+            const url = vdom.tags('ResourceURL').next().value;
+            o(url?.toString()).equals(
                 '<ResourceURL format="image/png" resourceType="tile" ' +
```

Comment:
Why do you prefer `url?.` to `url!` in tests?

Reply:
given `x = { a: 1} | null`

```
x = null
y = x.a!  // Runtime exception 
y = x?.a // y = undefined
```

inside of tests
```
o(x.a!).equals('foobar') // TypeError: Cannot read property 'a' of null
o(x?.a).equals('foobar') // expected undefined to equal "foobar"
```

and generated code

```
var x = null;
console.log(x.a);
console.log(x === null || x === void 0 ? void 0 : x.a);
```

So in general never use `!` unless you know for a fact that the thing is never going to be null, eslint also warns me about `!` usage as its not really good to use.